spec/services/zendesk_intake_service_spec.rb

```diff
@@ -59,8 +59,10 @@
     DatadogApi.configure do |c|
       c.enabled = true
       c.namespace = "test.dogapi"
+      c.env = "production"
```

Comment:
Why was this line added?

Reply:
Was it needed to prevent breakage of the test?

Reply:
yes! because the test_ticket tag depends on the environment, i had to change all the places where env was set in the test to be consistent otherwise the test failed. it was actually a bit of a challenge because i didn't realize i had to unset it in the after block